src/pretix/base/models/vouchers.py

```diff
@@ -163,7 +163,7 @@ def delete(self, using=None, keep_parents=False):
         super().delete(using, keep_parents)
         self.event.get_cache().delete('vouchers_exist')
 
-    def is_ordered(self) -> int:
+    def is_ordered(self) -> bool:
```

Comment:
Could you remove that change? :) This leads to a merge conflict, because as discussed I finally got rid of this method now. (Didn't push yesterday, my fault.)